flathub-external-data-checker/entrypoint.sh

```diff
@@ -7,6 +7,14 @@ fi
 
 detect_manifest() {
     repo=${1}
+    
+    # check if repo opted out
+    if [[ -f $repo/flathub.json ]]; then
+        jq -e '."disable-external-data-checker"' < $repo/flathub.json
+        if [ $? -eq 0 ]; then
```

Comment:
Alternatively, we can invert the value in `jq` expression. Then, if `jq` itself fails (and return non-zero due to error), we'll also abort the check (which is probably a good idea in case of e.g. invalid json in `flathub.json`).
```suggestion
        jq -e '."disable-external-data-checker" | not' < $repo/flathub.json
        if [ $? -ne 0 ]; then
```